### PR TITLE
work work. zug zug.

### PR DESCRIPTION
**Pull request by stephenmelnicki**
I did the thing, @tonybergeron !

work work

### CODE REVIEW

File: client/common/components/clock-face/clock-face.jsx [hunk 1]
```diff
@@ -66,4 +68,4 @@ export default class ClockFaceComponent extends Component {
 ClockFaceComponent.propTypes = {
   offset: PropTypes.string,
   location: PropTypes.string
-};
+}
```
Comment (stephenmelnicki) [inline client/common/components/clock-face/clock-face.jsx:71]: These changes were made by the automagic lint fixer.

File: client/dashboard/components/dashboard/dashboard.jsx [hunk 2]
```diff
@@ -1,50 +1,64 @@
...
-},{
+}, {
   name: 'New York',
   timezone: TimeZoneConstants.NEW_YORK
-},{
+}, {
   name: 'London',
   timezone: TimeZoneConstants.LONDON
-}];
+}]
```
Comment (stephenmelnicki) [inline client/dashboard/components/dashboard/dashboard.jsx:21]: More linter fixes.

File: package.json [hunk 3]
```diff
@@ -13,49 +13,55 @@
     "dbLocation": "mongodb://localhost/didactic-disco"
   },
   "scripts": {
-    "start": "NODE_ENV=development parallelshell 'npm run express-dev-server' 'npm run webpack-dev-server'",
+    "lint": "eslint --ext .jsx app",
+    "lint:fix": "npm run lint -- --fix",
+    "pretest": "npm run lint",
+    "test": "echo 'no tests available' && exit 1",
+    "bundle": "NODE_ENV=production webpack --config webpack/config.dist.js --progress",
```
Comment (stephenmelnicki) [inline package.json:19]: adding in some new commands here. these can be played around with as you see fit.

File: package.json [hunk 4]
```diff
@@ -13,49 +13,55 @@
...
     "eslint": "^2.9.0",
     "eslint-plugin-react": "^5.0.1",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "json-loader": "^0.5.4",
+    "nodemon": "^1.9.2",
     "parallelshell": "^2.0.0",
+    "postcss-loader": "^0.9.1",
+    "sass-loader": "^3.2.0",
+    "style-loader": "^0.13.1",
+    "webpack": "^1.13.0",
```
Comment (stephenmelnicki) [inline package.json:65]: I've moved everything that's not being used at run time (either client or server side) into the dev dependencies.

Comment (tonybergeron) [inline package.json:65]: the only thing that might need to be noted somewhere (and for your own knowledge) is if a CI server is in production mode and is building things in production, the npm install will default to only production installs as well.  

`npm --production=false install` needs to be used then in CI servers

**Pull request merged**